install

```diff
@@ -21,18 +21,22 @@ main() {
   #local exec_filename=oil.ovm-dbg
   local exec_filename=oil.ovm  # Should this go in configure output?
 
-  local dest="$PREFIX/bin/$exec_filename"
+  local BINDIR="${PREFIX}/bin"
```

Comment:
Local variables should be lower case.  Please revert all the renamings of local variables.

Document DESTDIR at the top of the file, since it's part of the user interface.